python/heterocl/compute_api.py

```diff
@@ -10,9 +10,12 @@
 from .types import Fixed, UFixed, Struct, dtype_to_str, dtype_to_hcl
 from .schedule import Stage
 from .debug import APIError
-from .dsl import if_, for_
+from .dsl import if_, for_, def_
 from .mutator import Mutator
 from .module import Module
+from .types import Type
+from . import util
+from itertools import product
```

Comment:
Double-check whether these are indeed used or not.

Reply:
They are used indeed in the bitcast API. `util` and `Type` are used to check the input destination datatype and bitwidth, `product` is used to iterate through the input tensor

Reply:
Please see [line 8 and 10](https://github.com/cornell-zhang/heterocl/blob/bbfdae4abbbe9268c65cdcad7b8d312412cf1248/python/heterocl/compute_api.py#L8). `util` and `types` are already imported. For `itertools` move it to the top along with `numbers` and `numpy`.